llvm/test/Transforms/GlobalOpt/global-opt-addrspace.ll

```diff
@@ -8,9 +8,9 @@
 @_ZM2C = internal addrspace(1) global %struct.FakeDeviceGlobal zeroinitializer, align 8
 @_ZL1C = internal addrspace(1) global %struct.FakeDeviceGlobal zeroinitializer, align 8
 
-@llvm.compiler.used = appending global [2 x ptr addrspace(4)] [ptr addrspace(4) addrspacecast (ptr addrspace(1) @_ZM2C to ptr addrspace(4)), ptr addrspace(4) addrspacecast (ptr addrspace(1) @_ZL1C to ptr addrspace(4))]
```

Comment:
I would introduce a separate test for this, instead of modifying this existing one